gormapp/gormapp.go

```diff
@@ -0,0 +1,95 @@
+package gormapp
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/fabric8-services/fabric8-env/application"
+	"github.com/fabric8-services/fabric8-env/environment"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+)
+
+type TXIsoLevel int8
+
+const (
+	// See https://www.postgresql.org/docs/9.3/static/sql-set-transaction.html
+
+	TXIsoLevelDefault TXIsoLevel = iota
+	TXIsoLevelReadCommitted
+	TXIsoLevelRepeatableRead
+	TXIsoLevelSerializable
+)
+
+var _ application.DB = &GormDB{}
+
+var _ application.Transaction = &GormTransaction{}
+
+func NewGormDB(db *gorm.DB) *GormDB {
+	return &GormDB{GormBase{db}, ""}
```

Comment:
Not sure what it gives us but in other services we also have this one - https://github.com/fabric8-services/fabric8-auth/blob/master/gormapplication/application.go#L56
Worth checking gorm docs.